shinysdr/i/network/webapp.py

```diff
@@ -188,10 +219,10 @@ def _put_plugin_resources(wcommon, client_resource):
     load_list_js = []
     mode_table = {}
     plugin_resources = Resource()
-    client_resource.putChild('plugins', plugin_resources)
+    client_resource.putChild(b'plugins', plugin_resources)
     for resource_def in getPlugins(_IClientResourceDef, shinysdr.plugins):
         # Add the plugin's resource to static serving
-        plugin_resources.putChild(resource_def.key, resource_def.resource)
+        plugin_resources.putChild(resource_def.key.encode(), resource_def.resource)
```

Comment:
This one definitely needs to be 'utf-8' as the plugin could put arbitrary characters in its names.

Reply:
Does that mean it also needs to be URL-encoded? Otherwise done.